mos-platform/nes-gtrom/prg-rom.ld

```diff
@@ -1,25 +1,27 @@
 INCLUDE _prg-rom-banked.ld
 
-PROVIDE(__prg_rom_size = 512);
```

Comment:
Here it still makes sense to use an extreme size as the default; the maximum seems preferable to the minimum. We should add a test for a smaller size instead; since this is auto-generated by Lua, we can rely on a test for one case to provide information on all similar cases.

Reply:
PTAL.